pkg/descriptions.json

```diff
@@ -441,6 +441,16 @@
     ]
   },
   {
+    "Name": "System.Data.DataSetExtensions",
+    "Description": "Provides extensions to form LINQ expressions and method queries against DataTable objects.",
+    "CommonTypes": [
+      "System.Data.DataTable",
```

Comment:
I don't see DataTable in the reference assembly, nor SortExpressionBuilder. 
 https://github.com/nchikanov/corefx/blob/52c2e03a2fad1856e9f7b767ea4cdc2c3f79df23/src/System.Data.DataSetExtensions/ref/System.Data.DataSetExtensions.cs